packages/gatsby-source-strapi/.gitignore

```diff
@@ -0,0 +1,161 @@
+# Logs
```

Comment:
This file should be able to be removed, the repo level ignores should cover everything. If that's not the case let's slim this down to not have all the generic ignores.